vanilla-todo/src/App.js

```diff
@@ -0,0 +1,86 @@
+import Component from "./Component.js";
+
+import Title from "./components/Title.js";
+import AddForm from "./components/AddForm.js";
+import TodoList from "./components/TodoList.js";
+
+class App extends Component {
+  initState() {
+    return {
+      todos: [],
+      editing: null,
+    };
+  }
+  html() {
+    return `
+      <header id="Title"></header>
```

Comment:
```suggestion
      <header id="Title">${Title()}</header>
```
Title 컴포넌트에는 별도의 상호작용이 없는 상태라서, 이렇게 함수를 실행하면 text를 반환하는 형태로만 관리해줘도 좋지 않을까요?